### ISSUE TITLE
acc is very bad

### ISSUE THREAD

**Issue by muyuuuu** [TheoT1/FW-AT-Adapt#2]
when I run `python train.py --mode=fw_adapt --dataset cifar10 --num_epochs=160 --ep_decay=60`

![image](https://user-images.githubusercontent.com/43681138/219953662-eaefd79b-1685-4f22-ba54-aac9b890dbe8.png)

I want to train my own model with FW-ADAPT ，so different model structure prevents me from using your pretrained model

**Comment by Jay-Roberts** [comment]
Hello! Are you able to train your model with the code base without adversarial training so we can compare the natural accuracy?

```
python train.py --mode=standard --dataset cifar10 --num_epochs=160 --ep_decay=60
```
